backend/specs/all-letters-match.spec.js

```diff
@@ -0,0 +1,41 @@
+import { createSolutionToBe, v2guess } from './utils/test-api-calls'
+
+describe('Guess feedback by letter', () => {
+
+    test('when guess letters are all present', async () => {
+        const WORDLE = 'abcde'
+        const GUESS = 'abcde'
+
+        await createSolutionToBe(WORDLE)
+
+        const expectedReponse = [
+            {
+                "letter": "a",
+                "position": 0,
```

Comment:
Yea, I guess it was a "defensive-programming" approach.  Like... what if someone messes with the payload before I get it, and I can't trust the order of the array?  I'm down to change it though.  Makes it more concise.

- [ ] Remove redundant position field in response entity

Reply:
I don't think it's possible for anyone to mess with the payload? In this instance, the API is creating this array and marshalling it to JSON on each guess. The only time you have to be careful about ordering is when something is coming from a database, otherwise it's likely just a problem with your API logic.

Reply:
Something about it made me more cozy knowing I was also attaching the position, but probably just paranoia and defensiveness.

Reply:
I've been there before